tests/config.py

```diff
@@ -37,8 +37,7 @@
 # The node image by iID known to the provider
 # NOTE(jhesketh): The libcloud OpenStack driver only supports finding images by
 #                 ID
-NODE_IMAGE_ID = os.getenv('NODE_IMAGE_ID',
-                          'e9de104d-f03a-4d9f-8681-e5dd4e9cede7')
+NODE_IMAGE = os.getenv('NODE_IMAGE', None)
```

Comment:
Just need to update the doc comment here please.

Reply:
Done